cmd/cluster/clusterDelete.go

```diff
@@ -132,7 +132,8 @@ func parseDeleteClusterCmd(cmd *cobra.Command, args []string) []*k3d.Cluster {
 
 		c, err := client.ClusterGet(cmd.Context(), runtimes.SelectedRuntime, &k3d.Cluster{Name: clusterDeleteCfgViper.GetString("name")})
 		if err != nil {
-			l.Log().Fatalf("failed to delete cluster '%s': %v", clusterDeleteCfgViper.GetString("name"), err)
+			l.Log().Infof("Deleting cluster '%s'", clusterDeleteCfgViper.GetString("name"))
+			return nil
```

Comment:
With no extra condition, this will return nil, even if there was an actual error e.g. in the runtime function that makes the "ClusterGet" fail.
```suggestion
			if errors.Is(err, client.ClusterGetNoNodesFoundError) {
				l.Log().Infof("No nodes found for cluster '%s', nothing to delete.", clusterDeleteCfgViper.GetString("name"))
				return nil
			}
```

Reply:
Hi @iwilltry42 ,
💯  and thanks for review! To make it compilable, I only had to add and `errors` namespace. 

```log
# k3d cluster delete -c ./local.yaml
INFO[0000] Using config file ./local.yaml (k3d.io/v1alpha3#simple) 
INFO[0000] No nodes found for cluster 'mucluster', nothing to delete. 
INFO[0000] No clusters found                            
```

Reply:
Yeah true, I couldn't add the import in 5he suggestion 👍